modules/wallet/admin/config.php

```diff
@@ -55,6 +55,8 @@
         $array_config['transaction_expiration_time'] = 0;
     }
 
+    $array_config['captcha_type'] = $nv_Request->get_string('captcha_type', 'post', '');
```

Comment:
@HQuang chỗ này không ổn em.
- get string là kém an toàn, dùng get_title
- `$array_config['captcha_type']` chưa được kiểm tra, lỡ submit lên giá trị xyz nào đó nó vẫn ghi nhận